### ISSUE TITLE
ORM maybe?

### ISSUE THREAD

**Issue by holazzer** [AllToBeNice/PersonalProject#1]
Why don't you try ORM? I think it is an awesome tool for both RDBMS as well as NoSQL. You can just use one class to handle the record and leave SQL statements to ORM. You may use sqlalchemy for any SQL Server supported, sqlite included.

**Comment by AllToBeNice** [comment]
Yes, you are right. The inconsistent style of SQL statements with python has troubled me for a long time.
P.S. I'm sorry for my late because I use Gmail.😅

**Comment by AllToBeNice** [comment]
Yes, you are right. The inconsistent style of SQL statements with python has troubled me for a long time.
P.S. I'm sorry for my late because I use Gmail.😅
